src/parser.ts

```diff
@@ -127,6 +133,7 @@ class SdpGrammar extends Grammar {
     this.addParser('a', RidLine.fromSdpLine);
     this.addParser('a', CandidateLine.fromSdpLine);
     this.addParser('a', SimulcastLine.fromSdpLine);
+    this.addParser('a', RidLine.fromSdpLine);
```

Comment:
Did you mean to put something else here instead of `RidLine` again?

Reply:
Yikes, yep.  Thanks.